src/ruleLoader.ts

```diff
@@ -43,23 +38,19 @@ export function loadRules(ruleOptionsList: IOptions[],
 
     for (const ruleOptions of ruleOptionsList) {
         const ruleName = ruleOptions.ruleName;
-        const enableDisableRules = enableDisableRuleMap.get(ruleName);
-        if (ruleOptions.ruleSeverity !== "off" || enableDisableRuleMap) {
```

Comment:
It's not necessary to check `ruleSeverity` here since `isEnabled()` will do that too.